src/agentscope/models/model.py

```diff
@@ -103,16 +105,21 @@ def image_urls(self) -> Sequence[str]:
         return self._image_urls
 
     @property
-    def raw(self) -> dict:
-        """Raw dictionary field."""
+    def raw(self) -> Any:
```

Comment:
The `Any` type is too free. It is recommended to give a warning if using this field and add another field named `dictionary` or `json` in `ModelResponse` to inherit the original `raw` field.

Reply:
The response from Gemini is in neither json nor string format, and we cannot ensure that responses from all model APIs can be turned into json or string in the future.
(In my view, since the response is from remote model services, we can assume that raw response is serializable.)